### PR TITLE
simple bot setup

### PR DESCRIPTION
**Pull request by Enubia**

### CODE REVIEW

**Review by stacysealky — COMMENTED** [review]
Good to merge or still WIP? @Enubia

**Comment by Enubia** [comment]
> Good to merge or still WIP? @Enubia

still WIP, only did the basic stuff

**Comment by Enubia** [comment]
ready for review @stacysealky

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,4 @@ dump.*
 
 server/db/seederCredentials.json
 v1doc.md
+.vscode/settings.json
```
Comment (stacysealky) [inline .gitignore:25]: we are adding this vscode settings to gitignore now?

**Review by stacysealky — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (Enubia) [reply .gitignore:25]: ya it's just personal setting from me because i have different setup in every project

Comment (stacysealky) [reply .gitignore:25]: ah gotcha! sounds good

**Pull request merged**